tripal/src/TripalPubLibrary/TripalPubLibraryBase.php

```diff
@@ -97,4 +97,30 @@ public function __construct(array $configuration, $plugin_id, $plugin_definition
     $this->fileretriever = $fileretriever;
   }
 
+  /**
+   * Validates whether XML is valid or not.
+   *
+   * @param string $xml
+   *   The XML to be checked.
+   *
+   * @return bool
+   *   Return TRUE if valid, FALSE if not valid.
+   */
+  protected function xmlIsValid(string $xml): bool {
+    $valid = TRUE;
+
+    libxml_use_internal_errors(TRUE);
```

Comment:
```suggestion
    // Enable user handling of errors so that exceptions are not
    // thrown when invalid XML is read.
    libxml_use_internal_errors(TRUE);
```